src/components/item/iterator.js

```diff
@@ -65,6 +83,13 @@ class ItemIterator extends Iterator {
         cat))
   }
 
+  isItemSortable() {
+    if (this.hasPositionColumn() && this.props.sort.column === 'added') {
```

Comment:
Better to return the result directly than the extra return true/false

Reply:
done